includes/polling/os/packetfluxpfr.inc.php

```diff
@@ -0,0 +1,41 @@
+<?php
```

Comment:
Please convert this to the new os format, https://docs.librenms.org/Developing/os/Initial-Detection/#os-discovery

Reply:
Took a run at this. The original code was produced by a co-worker. Let me know if I need to make further changes.

Reply:
So for this I would need to add a file to cover device discovery as referenced in the above link. The current code doesn't do discovery, and adding that feature is what is being requested?